### ISSUE TITLE
install namespace package

### ISSUE THREAD

**Issue by jmorten** [pypa/sample-namespace-packages#15]
Is it possible to have a `setup.py` file in the native directory that installs both pkg_a and pkg_b? If so, what would that look like?

something like
```
native
    |-- setup.py
    |-- pkg_a
        |-- example_pkg/a
        |-- setup.py
    |-- pkg_b
        |-- example_pkg/b
        |-- setup.py
```

Thank you.

**Comment by chrysle** [comment]
Yes, of course you can install all the namespace packages at once from outside, but they'd just behave like regular packages. The `pyproject.toml`/`setup.py` file would normally be in the project root directory, but of course you could
also place it under **native/**.

You wouldn't be able to import from the same namespace, though, instead:

```python
from native.pkg_a.example_pkg import a
from native.pkg_b.example_pkg import b
```
